cmake/ConfigureDoxygen.cmake

```diff
@@ -19,8 +19,14 @@ find_package(Doxygen
 	REQUIRED dot
 )
 
+
 if(DOXYGEN_FOUND)
-message("Use of DOXYGEN for developers: Please use this project's doxygen documentation style specified in ${CMAKE_CURRENT_SOURCE_DIR}/Documentation/doxymentation/StyleSheetOIP.")
+message("Use of DOXYGEN for developers: Refer to DoxygenHelp.md on how to use DOXYGEN within the OIP.")
+
+# Doxygen build options. 
+Option(DOXYGEN_OPTIONAL_COMMENTED_ONLY ON)
+Option(DOXYGEN_OPTIONAL_GENERATE_INTERNAL ON)
+Option(DOXYGEN_OPTIONAL_INCLUDE_EARLYBINDING ON)

```

Comment:
These are named differently in DoxygenHelp.md.

Reply:
Resolved: https://github.com/tumcms/Open-Infra-Platform/pull/104/commits/2f3ce53f07e3818cdd87584cabc425e275b1a4f5